### PR TITLE
Disable scenarios not supported for OCP v4.12 also for OCP v4.13

### PR DESCRIPTION
**Pull request by pmacik**
Signed-off-by: Pavel Macík <pavel.macik@gmail.com>

This PR is a dependency of https://github.com/openshift/release/pull/35524

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
  included if any changes are user facing
- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
- [ ] All acceptance test scenarios included in the PR which verifies a bugfix or a requested feature reported by a non-member are tagged with `@external-feedback` tag.
- [ ] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)

### CODE REVIEW

**Comment by pmacik** [comment]
@filariow PTAL

**Review by filariow — APPROVED** [review]

File: test/acceptance/features/supportExistingOperatorBackedServices.feature [hunk 1]
```diff
@@ -445,10 +448,10 @@ Feature: Support a number of existing operator-backed services out of the box
            """
     And File "/bindings/$scenario_id/password" exists in application pod
 
-  @disable-openshift-4.9
```
Comment (filariow) [inline test/acceptance/features/supportExistingOperatorBackedServices.feature:448]: This PR will still run checks on OpenShift 4.9, we should remove this tag after merging [openshift/release#35524](https://github.com/openshift/release/pull/35524).

Comment (filariow) [reply test/acceptance/features/supportExistingOperatorBackedServices.feature:448]: or force merging here with failing check on 4.9

Comment (pmacik) [reply test/acceptance/features/supportExistingOperatorBackedServices.feature:448]: This scenario is a flaky one on 4.9, that is why it has been disabled, luckily it passed this time.

**Review by filariow — APPROVED** [review]

**Pull request merged**